docs/installation.rst

```diff
@@ -292,9 +292,7 @@ or from within the uncompressed source directory::
 Building on Windows
 ^^^^^^^^^^^^^^^^^^^
 
-We don't recommend trying to build on Windows. It is a maze of twisty
```

Comment:
Just to be clear - so you think that this PR improves the situation enough that it is now viable for users to compile from source by themselves?

Reply:
While installing Visual Studio is not exactly easy, once that is done, compiling is as difficult as running the script. It is by no means a maze anymore. I have not had to do any extra configuration on my new laptop whatsoever.

That said, when writing it I was thinking more about developers (contributing PRs) than users. AFAIK most Windows users aren't used to compiling code at all, so perhaps it would be best to write it as "We recommend you use prebuilt wheels from PyPI. If you wish to compile Pillow manually, you can use the build scripts and notes ..."

Reply:
@radarhere any comments on the current wording? Okay to merge?

Reply:
Wording is fine, just wanted to check that we knew that we were stepping up our level of commitment to building from source on Windows.